Tella/Domain/Entity/Report/Project.swift

```diff
@@ -30,19 +25,16 @@ class Server : Hashable {
          projectId: String? = nil,
```

Comment:
I think `activatedMetadata`, `backgroundUpload`, `projectId`, `slug`, `autoUpload` and  `autoDelete` are not used